doc/Makefile

```diff
@@ -0,0 +1,169 @@
+# Time-stamp: <2017-12-07 11:11:52 kmodi>
+
+# Makefile to generate use-package doc site
+
+EMACS ?= emacs
+EMACS_exists := $(shell command -v $(EMACS) 2> /dev/null)
+ifeq ("$(EMACS_exists)","")
+	EMACS := /tmp/emacs/bin/emacs
+endif
+
+# EMACS_BIN_SOURCE and EMACS_VERSION are used later in the vcheck rule
+# only if EMACS_exists has evaluated to "".
+EMACS_BIN_SOURCE ?= https://github.com/npostavs/emacs-travis/releases/download/bins
+EMACS_VERSION ?= 25.2
```

Comment:
You should use 25.3, there's never any reason to use 25.2 when 25.3 is available.

Reply:
Relic of the past, will fix it. Thanks!